clean_build.sh

```diff
@@ -1,32 +1,74 @@
 #!/bin/sh
+
 cd "$(dirname "$0")" || exit 1
+
 # some environments have cmake v2 as 'cmake' and v3 as 'cmake3'
 # check for cmake3 first then fallback to just cmake
-B_CMAKE=`type cmake3 2>/dev/null`
-if [ $? -eq 0 ]; then
-    B_CMAKE=`echo "$B_CMAKE" | cut -d' ' -f3`
+
+B_CMAKE=$(command -v cmake3 2>/dev/null)
+if [ "$?" -eq 0 ]; then
+    # Continue, cmake3 exists in $PATH.
```

Comment:
Obvious comment, no need for it.

I think it maybe makes sense to put all of this in a loop to check `cmake3` and `cmake` in that sequence and exit the loop if the command is found.

Reply:
That's been removed - the comment - but I think we should really think pragmatically about changing this into a loop. What benefits does it bring? If we're just checking for either executable, a conditional works fine IMO. Thoughts?

Reply:
Fixed in 250072e9.